spec/middleware/authentication_spec.rb

```diff
@@ -16,6 +16,15 @@
 
         expect(status).to eq(200)
       end
+
+      it 'allows the request if given a correct password header and a Passwords header' do
+        app = lambda { |env| [200, {'Content-Type' => 'application/json'}, Array.new] }
+        middleware = Napa::Middleware::Authentication.new(app)
+        env = Rack::MockRequest.env_for('/test', {'HTTP_PASSWORD' => 'foo', 'HTTP_PASSWORDS' => 'foo,bar'})
```

Comment:
Line is too long. [107/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.